jib-gradle-plugin/src/integration-test/java/com/google/cloud/tools/jib/gradle/JibPluginIntegrationTest.java

```diff
@@ -254,4 +261,23 @@ public void testDockerContext() throws IOException, InterruptedException {
               "Export Docker context failed because cannot clear directory"));
     }
   }
+
+  /**
+   * Asserts that the creation time of the simple test project is set. If the time parsed from the
+   * {@code docker inspect} command occurs before the specified time (i.e. if it is 1970), then the
+   * assertion will fail.
+   *
+   * @param before the specified time to compare the resulting image's creation time to
+   * @throws IOException if the {@code docker inspect} command fails to run
+   * @throws InterruptedException if the {@code docker inspect} command is interrupted
+   */
+  private static void assertSimpleCreationTimeIsAfter(Instant before)
+      throws IOException, InterruptedException {
+    String inspect =
+        new Command("docker", "inspect", "gcr.io/jib-integration-testing/simpleimage:gradle").run();
+    int beginQuote = inspect.indexOf("\"Created\": \"") + 12;
```

Comment:
This might be subject to break on different versions of `docker inspect` (different whitespacing for example). Maybe a regex match would be safer

Reply:
Oh might want to use something like this: https://stackoverflow.com/a/32705298/10047775